### DISCUSSION TITLE
An older version of xdm is already running

### DISCUSSION THREAD

**Discussion by abhishek-chauhan8** [subhra74/xdm#904] [General]
When opening xdm in linux, a pop-up window is showing that "an older version of xdm is already running" . Done everything whether it's uninstall or reinstall the same or other version, nothing works. Also deleted pre-existing folder of xdm video downloads and also removed extension from Google Chrome Browser. Is any other way to resolve the issue.

**Comment by NormB3** [comment]
I have the same issue on my Windows 10 desktop. I can not get it to go away no matter what I do. The 'Older Version' is from 2020.
